public/home.html

```diff
@@ -46,16 +46,25 @@
   </nav>
 
   <!-- Hero section -->
-  <div class="hero d-flex flex-column align-items-center justify-content-around">
+  <!-- <div class="hero d-flex flex-column align-items-center justify-content-around">
     <h2 style="color: #fff;">Book Trading Club</h2>
     <i class="fa fa-book fa-3x fa-inverse" aria-hidden="true"></i>
+  </div> -->
+
+  <div class="jumbotron jumbotron-fluid hero text-center">
+    <div class="container">
+      <h2><i class="fa fa-book fa-2x fa-inverse" aria-hidden="true"></i></h2>
+      <h1 class="display-4">Book Trading Club</h1>
+      <p class="lead">Have old books you want to trade and looking for new books to read?</p>
+      <a href="#" class="btn btn-outline-light btn-lg">Sign Up Now!</a>
```

Comment:
We should remove the sign up now box for an authenticated user. Let's keep it on the login page, but remove it for home page.